Data/4_Data.csproj

```diff
@@ -5,12 +5,19 @@
     <LangVersion>8</LangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Migrations\20200903084647_AddDescriptionToHighlight.cs" />
+    <Compile Remove="Migrations\20200903084647_AddDescriptionToHighlight.Designer.cs" />
+    <Compile Remove="Migrations\20200904100315_ChangedHighLightDescriptionToRequired.cs" />
+    <Compile Remove="Migrations\20200904100315_ChangedHighLightDescriptionToRequired.Designer.cs" />
+  </ItemGroup>
```

Comment:
These are still leftovers from testing your migrations. This code block can be removd.

Reply:
If you have fixed this comment, click on "Resolve conversation"!

Reply:
@Brend-Smits Isn't it already done? It shows "Outdated" next to the file name. :)

Reply:
Nope, see: https://gyazo.com/85a5a5329ddc447f985727d1d3457aba
Bottom left. I can of course do it myself but I'm trying to teach you all the processes! ^.^